docs-src/security-model.md

```diff
@@ -72,14 +72,12 @@ readable by most roles, so instead we'll focus on write access for this model.
 The extra configuration options are not possible to control with RBAC. Instead,
 they will be controlled with configuration fields on GatewayClasses:
 
-* **allowedGatewayNamespaceSelector**: This field is a selector of namespaces
-  that Gateways can use this GatewayClass from. This is a standard Kubernetes
-  LabelSelector, a label query over a set of resources. The result of
-  matchLabels and matchExpressions are ANDed. Controllers must not support
-  Gateways in namespaces outside this selector. An empty selector (default)
-  indicates that Gateways can use this GatewayClass from any namespace. This
-  field is intentionally not a pointer because the nil behavior (no namespaces)
-  is undesirable here.
+* **allowedGatewayNamespaceSelector**: AllowedGatewayNamespaceSelector is a
+  selector of namespaces that Gateways of this class can be created in. Gateway
+  validation will prevent Gateways specifying this class from being created
```

Comment:
Again, specifying  validation webhook would be more clearer here as our users do often confuse validation with openapi schema validation.

Reply:
Updated these docs to reflect the new proposal